composer.json

```diff
@@ -13,6 +13,9 @@
         {"name": "Marco Pivetta", "email": "ocramius@gmail.com"}
     ],
     "config": {
+        "platform": {
+            "php": "7.1.3"
```

Comment:
I picked this version because it is require by recent versions of Psalm, I hope this is not too much of a big deal.

Reply:
It is described as "root-only" so in context of projects using ORM as a dependency this shouldn't be a problem. But I'm not so sure for the ORM-build itself which may fetch older packages listed in composer's `require`.

Reply:
Removing it makes it really harder to use composer and be sure the `composer.lock` you generate is usable on build jobs that use old versions of php (e.g. if you install a version of a package that requires 7.2)

Reply:
I hope this won't give us any trouble in the future. :crossed_fingers: